priam/src/test/java/com/netflix/priam/resources/BackupServletTest.java

```diff
@@ -45,6 +49,7 @@
     private @Mocked Provider<AbstractBackupPath> pathProvider;
     private @Mocked CassandraTuner tuner;
     private @Mocked SnapshotBackup snapshotBackup;
+    private @Mocked IncrementalBackup incrmentalBkup;
```

Comment:
incrmentalBkup ---> incrmentalBackup (be similar to the previous terminology of snapshotBackup as shown above)